add Bailing in provider options

codegrandpa opened the pull request:
### Provider Details

API Type: OpenAI-compatible
Base URL: https://api.tbox.cn/api/llm/v1/chat/completions
SDK: @ai-sdk/openai-compatible
Documentation: https://alipaytbox.yuque.com/sxs0ba/ling/intro
Environment Variable:BAILING_API_TOKEN

### MODELS
Ling-1T
Ring-1T

<img width="2560" height="396" alt="image" src="https://github.com/user-attachments/assets/fae8346d-4a4b-4f0b-b588-654f5f28191c" />

Closes #4676 on sst/opencode

Commit:
Add new provider: bailing

codegrandpa committed:
Merge branch 'sst:dev' into dev

Commit:
add log

Commit:
Merge remote-tracking branch 'origin/dev' into dev

codegrandpa committed:
Merge branch 'sst:dev' into dev

Commit:
change logo

Commit:
Merge remote-tracking branch 'origin/dev' into dev

rekram1-node commented:
can you make the logo.svg a square and have no color

see other logo.svgs for reference

Commit:
update logo

codegrandpa commented:
> can you make the logo.svg a square and have no color
> 
> see other logo.svgs for reference

I’ve updated the logo.svg to be square and removed the color, following the other logos as reference Please take another look
